src/main/java/spotify/api/impl/BrowseApiRetrofit.java

```diff
@@ -83,6 +89,34 @@ public PlaylistSimplifiedPaging getCategoryPlaylists(String categoryId, Map<Stri
         }
     }
 
+    @Override
+    public Paging<PlaylistSimplified> getCategoryPlaylistsUnwrapped(String categoryId, Map<String, String> options) {
```

Comment:
I don't really like the "unwrapped" in the function name. It's added for historic reason (i.e. a deprecated function with the same name) but the API user shouldn't need to know that. And if we're phasing out the deprecated function we're stuck with the unwrapped in the function name.

Reply:
To be honest I don't like it either. However, the methods need to be differentiated somehow, so a new name is needed. These are the ones I could come up with:

- `getCategoryPlaylistPaging`
- `getCategoryPlaylistsPage`
- `getPlaylistsOfCategory`

What do you think?

Reply:
`getCategoryPlaylistPaging` seems good, as it is also consistent with other method names that return `Paging`

Reply:
Thanks, updated the methods' names.